src/oss/confluent.sh

```diff
@@ -665,7 +672,7 @@ validate_java_version() {
     # 10 -> java version "10" 2018-03-20
     # 10.0.1 -> java version "10.0.1" 2018-04-17
     # We need to match to the end of the line to prevent sed from printing the characters that do not match
-    local java_version=$(java -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
+    local java_version=$(${JAVA} -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
```

Comment:
there are multiple references to `java` directly in the rest of the file. shouldn't those now use `${JAVA}`?

Reply:
@wicknicks the only other references I see are in calls to `lsof | grep`; do you see any others?